### PR TITLE
updating press kit link

### PR DESCRIPTION
**Pull request by pcmccarron**
pointing to the brand page instead of a zip file.

### CODE REVIEW

File: website/components/footer/index.jsx [hunk 1]
```diff
@@ -21,9 +21,7 @@ export default function Footer({ openConsentManager }) {
           <Link href="/security">
             <a>Security</a>
           </Link>
-          <Link href="/files/press-kit.zip">
-            <a>Press Kit</a>
-          </Link>
+          <a href="https://www.hashicorp.com/brand">Press Kit</a>
```
Comment (jkirschner-hashicorp) [inline website/components/footer/index.jsx:24]: I'm not familiar with why the footer wraps `<a>` tags that are links in `<Link>` tags, but just noting that inconsistency. Maybe it should be like this to be consistent?

```suggestion
          <Link href="https://www.hashicorp.com/brand">
            <a>Press Kit</a>
          </Link>
```

**Review by jkirschner-hashicorp — APPROVED** [review]
LGTM

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/571673.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit eaf28e38dd23ea4c9f4efd5419babc202b941c53 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit eaf28e38dd23ea4c9f4efd5419babc202b941c53 onto `release/1.11.x` succeeded!

**Pull request merged**